### ISSUE TITLE
Reverse direction if swiping/rotating too far accidentally 

### ISSUE THREAD

**Issue by supercheetah** [8VIM/8VIM#558]
**Is your feature request related to a problem? Please describe.**
No, at least I don't think so.

**Describe the solution you'd like**
I'd like to be able to "rotate" backwards from the direction I started going in for when I accidentally go too far, or just change my mind about which letter to use.  For example, let's say I start gesturing to get a "v", but I accidentally go too far, and am on "w" in the bottom sector.  I should be to reverse direction back into the left sector to get the "v", and then go back to the center to complete it.

**Describe alternatives you've considered**
To stop, and backspace over the space that's automatically put in there, which is just slow, and extra disruptive.

**Comment by ManDay** [comment]
Hello, there are several points to this:

- It is not possible, because reversing is already used to obtain variants of a letter (in layouts which have multiple "layers" - see for example French or German)
- With a bit more experience with 8vim you will notice that this is not necessary. Correcting a "too many turns" mistake is something that only happens to you as a beginner. You will quickly stop making that error when you have some practice.
- The "correct" thing to do when you noticed that you made a turn too many, is to just let go. The standard layouts are designed such that this will cause an "abort", because nothing is bound to gestures where you lift your finger after a turn, without returning to the center.

**Comment by MaethorNaur** [comment]
Yes, the auto space is an "issue" that I was thinking about. however going back when you missed a movement to undo is not possible as it could be a movement as @ManDay said
